helm/hpcc/templates/roxie.yaml

```diff
@@ -15,14 +15,16 @@ data:
       topologyServers: "{{ .toponame }}:{{ .topoport }}"
       resolveLocally: false
 {{- include "hpcc.generateLoggingConfig" (dict "root" .root "me" .me) | indent 6 }}
-{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" (list "storage" "ecl" "ecl-user" ) ) | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" .secretsCategoriesUser ) | indent 6 }}
     global:
 {{ include "hpcc.generateGlobalConfigMap" .root | indent 6 }}
 {{- end -}}
 
 {{ range $roxie := $.Values.roxie -}}
 {{- if not $roxie.disabled -}}
-{{- $commonCtx := dict "root" $ "me" $roxie }}
+{{- $secretsCategoriesUser := list  "storage" "ecl" "ecl-user" }}
+{{- $secretsCategoriesSystem := list "system" "ecl" "storage" }}
```

Comment:
I don't think it makes sense for these to differ - the list provided to the vault config should match the volume mounts.